### PR TITLE
TroyeGuo.KoodoReader version 1.4.0.811

### PR DESCRIPTION
**Pull request by superusercode**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/44424)

### CODE REVIEW

**Comment by superusercode** [comment]
This worked fine for me locally.

**Comment by Trenly** [comment]
This is what I get on the sandbox environment
image

File: manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: TroyeGuo.KoodoReader
+PackageVersion: 1.4.0.811
```
Comment (Trenly) [inline manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.installer.yaml:5]:
```suggestion
PackageVersion: 1.4.0
```
This must match the version shown in control panel. Don't forget to update the version folder name when this changes

File: manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TroyeGuo.KoodoReader
+PackageVersion: 1.4.0.811
```
Comment (Trenly) [inline manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.4.0
```

File: manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: TroyeGuo.KoodoReader
+PackageVersion: 1.4.0.811
```
Comment (Trenly) [inline manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.yaml:5]:
```suggestion
PackageVersion: 1.4.0
```

File: manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: TroyeGuo.KoodoReader
+PackageVersion: 1.4.0.811
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/troyeguo/koodo-reader/releases/download/v1.4.0/Koodo-Reader-1.4.0.exe
+  InstallerSha256: 797149FFD50C7CBD31DBC077D1CB0726F715AA5CD812EB6FC32713139C973A92
```
Comment (Trenly) [inline manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.installer.yaml:10]:
```suggestion
  InstallerSha256: 797149FFD50C7CBD31DBC077D1CB0726F715AA5CD812EB6FC32713139C973A92
  Scope: machine
  ElevationRequirement: elevationRequired #This should fix the unattended error
```

File: manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+PackageIdentifier: TroyeGuo.KoodoReader
+PackageVersion: 1.4.0.811
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/troyeguo/koodo-reader/releases/download/v1.4.0/Koodo-Reader-1.4.0.exe
+  InstallerSha256: 797149FFD50C7CBD31DBC077D1CB0726F715AA5CD812EB6FC32713139C973A92
+ManifestType: installer
+ManifestVersion: 1.0.0
```
Comment (Trenly) [inline manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.installer.yaml:12]:
```suggestion
ManifestVersion: 1.1.0 # For the elevation requirement key
```

File: manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+PackageIdentifier: TroyeGuo.KoodoReader
+PackageVersion: 1.4.0.811
+PackageLocale: en-US
+Publisher: Troye Guo
+PackageName: Koodo Reader
+License: AGPL-3.0 License
+ShortDescription: A cross-platform ebook reader
+ManifestType: defaultLocale
+ManifestVersion: 1.0.0
```
Comment (Trenly) [inline manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.locale.en-US.yaml:12]:
```suggestion
ManifestVersion: 1.1.0
```

File: manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.yaml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: TroyeGuo.KoodoReader
+PackageVersion: 1.4.0.811
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.0.0
```
Comment (Trenly) [inline manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.yaml:8]:
```suggestion
ManifestVersion: 1.1.0
```

File: manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.locale.en-US.yaml [hunk 8]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TroyeGuo.KoodoReader
+PackageVersion: 1.4.0.811
+PackageLocale: en-US
+Publisher: Troye Guo
```
Comment (Trenly) [inline manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.locale.en-US.yaml:7]:
```suggestion
Publisher:  App by Troye
```
This must match the entry in control panel

File: manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.locale.en-US.yaml [hunk 9]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TroyeGuo.KoodoReader
+PackageVersion: 1.4.0.811
+PackageLocale: en-US
+Publisher: Troye Guo
+PackageName: Koodo Reader
+License: AGPL-3.0 License
```
Comment (Trenly) [inline manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.locale.en-US.yaml:9]:
```suggestion
License: AGPL-3.0 License
LicenseUrl: https://github.com/troyeguo/koodo-reader/blob/master/LICENSE
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by superusercode** [comment]
> This is what I get on the sandbox environment image

I think this must be a successful install code? I can't find equivalents documented in the scoop and chocolatey versions of this application.

**Comment by superusercode** [comment]
Scratch that, looking up that error code seems to be related to incorrect memory use by the program. Not sure if this is on the VM or the installer itself.

**Comment by Trenly** [comment]
> Scratch that, looking up that error code seems to be related to incorrect memory use by the program. Not sure if this is on the VM or the installer itself.

When I install as admin, works just fine. Without admin, I get that error code, which is why I put the suggestions

**Comment by superusercode** [comment]
Is it still failing to the same error code?

**Comment by Trenly** [comment]
> Is it still failing to the same error code?

I can’t tell. Seems to work for me. You could try switching the architecture to x64 and see if that works?

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**